src/alfasim_sdk/_internal/constants.py

```diff
@@ -418,6 +418,17 @@ class PumpViscosityModel(Enum):
     AnsiHi2010 = "ansihi_2010"
 
 
+class PumpThermalEfficiencyModel(Enum):
+    """
+    Defines the pump thermal efficiency model
+        - ``Constant``: An user-defined constant value is used to define to transform the ESP mechanical energy to heat.
+        - ``PumpEfficiencyCurve``: The ESP efficiency curve is used to transform the ESP mechanical energy to heat.
+    """
+
+    Constant = "constant"
+    PumpEfficiencyCurve = "pump_eff_curve"
```

Comment:
Use the long name. it is clearer, and that is the value that ends in alfacase files.
```suggestion
    PumpEfficiencyCurve = "pump_efficiency_curve"
```

Reply:
Is this change necessary? We already have ``PumpEfficiencyCurve = "efficiency_curve_based"`` at the master.

Reply:
You are right.
Probably using the one from master is better.